components/title/multisize/src/index.scss

```diff
@@ -0,0 +1,60 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$c-title-multisize-primary: #a81c4f !default;
+$c-title-multisize-secondary: $c-gray !default;
+
+$ff-roboto-sans-serif: 'Roboto', sans-serif; // Will be moved to nc-theme-basic
+
+$fw-title-multisize-pre-title: 500 !default;
+$fw-title-multisize-main-title: 500 !default;
+$fw-title-multisize-post-title: $fw-regular !default;
+
+$fz-title-multisize-pre-title: $fz-h2 !default;
+$fz-title-multisize-main-title: 48px !default;
+$fz-title-multisize-post-title: $fz-h2 !default;
+
+$maw-phone: 480px !default; // Will be moved to nc-theme-basic
+
+$p-h-title-multisize-between-elements: $p-h-small;
+
+$phone: 'screen and (max-width: $maw-phone)'; // Will be moved to nc-theme-basic
```

Comment:
rather than use this mq, i would use theme-basic mq:
https://github.com/SUI-Components/theme-basic/blob/master/src/utils/_breakpoints.scss

also, theme-basic defined breakpoints:
https://github.com/SUI-Components/theme-basic/blob/master/src/_settings.scss#L137

for instance:

`@include media-breakpoint-down(s) {
    align-self: flex-start;
    padding: 0;
    text-align: left;
      }`